### PR TITLE
Build Tools: Update Prometheus and Grafana

### PR DESCRIPTION
**Pull request by markmandel**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug

/kind cleanup

> /kind documentation
> /kind feature
> /kind hotfix

**What this PR does / Why we need it**:

Updated the install instructions for our dev tooling to match PR #1850

**Which issue(s) this PR fixes**:

Related to #1850 

**Special notes for your reviewer**:

N/A

### CODE REVIEW

File: build/Makefile [hunk 1]
```diff
@@ -447,10 +447,10 @@ setup-prometheus: PVC ?= true
 setup-prometheus: PV_SIZE ?= 64Gi
 setup-prometheus: SCRAPE_INTERVAL=30s
 setup-prometheus:
-	$(DOCKER_RUN) helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+	$(DOCKER_RUN) helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 	$(DOCKER_RUN) helm repo update
-	$(DOCKER_RUN) helm upgrade prom stable/prometheus \
-		--install --wait --namespace metrics --create-namespace \
+	$(DOCKER_RUN) helm upgrade --install --wait prom prometheus-community/prometheus --version 11.16.2 \
```
Comment (roberthbailey) [inline build/Makefile:452]: Was there a reason you moved the `--install` and `--wait` flags before the name of the chart? Now there are some flags before the name and some after, which is odd (same question below as well).

File: build/Makefile [hunk 2]
```diff
@@ -464,14 +464,13 @@ setup-grafana: PVC ?= true
...
-	$(DOCKER_RUN) helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+	$(DOCKER_RUN) helm repo add grafana https://grafana.github.io/helm-charts
 	$(DOCKER_RUN) helm repo update
 	$(DOCKER_RUN) kubectl apply -f $(mount_path)/build/grafana/
-	$(DOCKER_RUN) helm upgrade grafana stable/grafana \
-		--install --wait --namespace metrics \
-		--set persistence.enabled=$(PVC),server.persistentVolume.size=$(PV_SIZE) \
-		--set adminPassword=$(PASSWORD) -f $(mount_path)/build/grafana.yaml
+	$(DOCKER_RUN) helm upgrade --install --wait grafana grafana/grafana --version=5.7.10 \
+					--namespace metrics --create-namespace \
```
Comment (roberthbailey) [inline build/Makefile:471]: The indentation here seems off (and not consistent with how it's done for the prometheus command above).

**Review by roberthbailey — COMMENTED** [review]
Just some minor formatting nits.

File: build/Makefile (hunk 1 above)
Comment (markmandel) [reply build/Makefile:452]: No real reason, was just copy pasting the referenced PR. I can switch it around.

Comment (markmandel) [reply build/Makefile]: Fixed!

File: build/Makefile (hunk 2 above)
Comment (markmandel) [reply build/Makefile]: Fixed!

**Review by roberthbailey — APPROVED** [review]

**Pull request merged**